templates/src/objective-c/BoringSSL-GRPC.podspec.template

```diff
@@ -70,7 +70,7 @@
 
   Pod::Spec.new do |s|
     s.name     = 'BoringSSL-GRPC'
-    version = '0.0.33'
+    version = '0.0.34'
     s.version  = version
     s.summary  = 'BoringSSL is a fork of OpenSSL that is designed to meet Google\'s needs.'
     # Adapted from the homepage:
```

Comment:
Can you try add the include files pattern in line 152 to include sub directories?
Add something like `'src/include/openssl/**/*.h'`

Reply:
That fixed that, but looks like there are other issues now :(

Reply:
Looks like path to [err_data.c](https://github.com/google/boringssl/blob/master/gen/crypto/err_data.c) changed, can you update it accordingly?

Reply:
Thanks! That fixed it